### DISCUSSION TITLE
Joining team

### DISCUSSION THREAD

**Discussion by JuanTamadski** [ROM-builders/temporary#13944] [Q&A]
Hi, can i join the team

Github: https://github.com/JuanTamadski
Telegram: @JuanTamadski

Successful builds:
Project blaze 1.4(below) and 1.5 EOL
https://cirrus-ci.com/task/4581764399104000

AICP s12.1
https://cirrus-ci.com/build/5798741461434368

Sources:
Z01R
Device tree - https://github.com/JuanTamadski/android_device_asus_Z01R
Kernel tree - https://github.com/bluehomewu/kernel_z01r
Vendor tree - https://github.com/yograjfire18/vendor_asus

**Comment by Apon77** [comment]
please check #team instruction in telegram

**Reply by JuanTamadski** [reply]
Already done that. Let me know if I missed something.

**Reply by Apon77** [reply]
> Already done that. Let me know if I missed something.

you missed #learn instruction said in #team
